Source/Wit/Public/Wit/TTS/WitTtsSpeaker.h

```diff
@@ -7,16 +7,14 @@
 
 #pragma once
 
-#include "TTS/Cache/Memory/TtsMemoryCache.h"
-#include "TTS/Cache/Storage/TtsStorageCache.h"
-#include "Wit/TTS/WitTtsService.h"
+#include "WitTtsExperience.h"
 #include "GameFramework/Actor.h"
 #include "WitTtsSpeaker.generated.h"
 
 class UAudioComponent;
 
 /**
- * The base class of TtsSpeaker
+ * Represents a speaker
  */
 UCLASS( ClassGroup=(Meta) )
 class WIT_API AWitTtsSpeaker : public AActor
```

Comment:
I feel better to call this  TtsSpeaker, if this may conflict with engine's class, we can also call it something like: VoiceSDKTtsSpeaker.